apps/docs/package.json

```diff
@@ -20,7 +20,7 @@
     "@docusaurus/preset-classic": "2.4.1",
     "@mdx-js/react": "^1.6.22",
     "@orama/plugin-docusaurus": "^1.2.4",
-    "@stylexjs/stylex": "0.4.1",
+    "@stylexjs/stylex": "0.5.0-beta.1",
```

Comment:
Typically we don't use beta channels for other projects, and might instead publish canaries using SHA (or combination of SHA and date) suffixes if a build is being made available for early testing.

Reply:
I'm still trying to figure out how to do a SHA hash. 

I didn't publish a `beta` yet, just a couple of `alpha`s.

Reply:
> I'm still trying to figure out how to do a SHA hash.

I can't remember if we automatically do anything for React etc, but in my own projects I've just taken 8-9 chars from the git hash at the HEAD commit being canary-published since no extra commit is added for canary releases.

Reply:
Oh! That's such a simple solution. Thanks! I'll do that from next time.